src/LegendList.tsx

```diff
@@ -442,7 +460,18 @@ const LegendListInner: <T>(props: LegendListProps<T> & { ref?: ForwardedRef<Scro
         }, []);
 
         const handleScroll = useCallback(
-            (event: { nativeEvent: { contentOffset: { x: number; y: number } } }, fromSelf?: boolean) => {
+            (
+                event: {
+                    nativeEvent: NativeScrollEvent;
+                },
+                fromSelf?: boolean,
+            ) => {
+                // in some cases when we set ScrollView contentOffset prop, there comes event from with 0 height and width
+                // this causes blank list display, looks to be Paper implementation problem
+                // let's filter out such events
+                if (event.nativeEvent?.contentSize?.height === 0 && event.nativeEvent.contentSize?.width === 0) {
```

Comment:
@jmeistrich When using LegendList on the old architecture, I was getting wrong contentSize and contentOffset equal zero. This code detects erroneous event and filters it out.